packages/gce-disk-expand/src/expandfs-lib.sh

```diff
@@ -111,6 +111,8 @@ parted_needresize() {
     return 1
   fi
 
+  udevadm settle
```

Comment:
this should be on line 113 of `src/expandfs-lib.sh`, right after the parted command. we might find out we need it in other places too, depending on how testing goes.